project/Dependencies.scala

```diff
@@ -97,16 +97,18 @@ object Dependencies {
     excludeKms,
     excludeBigQuery,
     excludeCloudBilling,
-    excludeBouncyCastle,
```

Comment:
this was in `workbench-google2`, which brings in 1.70. And we happen to be excluding them here, which was why we are still bringing 1.67 from other libraries